test/infrastructure/docker/third_party/forked/loadbalancer/const.go

```diff
@@ -16,8 +16,14 @@ limitations under the License.
 
 package loadbalancer
 
-// Image defines the loadbalancer image:tag
-const Image = "kindest/haproxy:v20210715-a6da3463"
```

Comment:
Should we prefix those contants with `Default` ?

Reply:
`Image` never changes, so I think we should probably leave that without the `Default` prefix? But definitely think that's a good idea for the other two.

Reply:
Yes, that makes sense.